sound/soc/intel/boards/sof_sdw.c

```diff
@@ -935,6 +958,13 @@ static int mc_probe(struct platform_device *pdev)
 
 	dmi_check_system(sof_sdw_quirk_table);
 
+	if (quirk_override != -1) {
```

Comment:
Should this be ``if (quirk_override >= 0)`` in case quirk_override is set to an invalid value by accident?

Reply:
it's the same code as in bytcr_rt5640/51, it's not something new. It doesn't really matter if the value is positive or not, only the bits that are set will be used. What we also discussed on alsa-devel is that a zero value is nice to remove all quirks.

Let's keep the same code as before.